dds/DCPS/DataWriterImpl_T.h

```diff
@@ -358,11 +358,11 @@ namespace OpenDDS {
         if (key_marshaled_size_) {
           effective_size = key_marshaled_size_;
         } else {
-          if (cdr && !Serializer::use_rti_serialization()) {
+          if (cdr) {
```

Comment:
This block should be removed because the block on line 365 will add the `cdr_header_size`.

Reply:
Fixed  - thanks.